content/livesync/models/index.textile

```diff
@@ -61,13 +61,13 @@ h3(#options). ClientOptions
 
 In addition to the underlying Ably realtime client, you can provide a number of other "@ClientOptions@":/api/realtime-sdk/types#client-options to configure the behavior of the Models SDK:
 
-- @syncOptions@ := is used to configure how the model state is synchronised via the "Sync function":#sync.
-- @historyPageSize@ := is the limit used when "querying for paginated history":/livesync/models/models#history used to subscribe to changes from the correct point in the channel.
-- @messageRetentionPeriod@ := is the message retention period configured on the Ably channel. This is used to determine whether the model state can be brought up to date from "message history":/livesync/models/models#history rather than via a re-sync.
-- @retryStrategy@ := defines a retry strategy to use if calling the "sync function":/livesync/models/models#sync throws an error.
-- @eventBufferOptions@ := used to configure the in-memory sliding-window buffer used for reordering and deduplication.
-- @optimisticEventOptions@ := is used to configure how "optimistic events":/livesync/models/models#optimistic are applied.
-- @logLevel@ := configures the log level used to control the verbosity of log output. One of @fatal@, @error@, @warn@, @info@, @debug@, or @trace@.
+- @syncOptions@ := Used to configure how the model state is synchronised via the "sync function":#sync.
```

Comment:
With the current docs (and the docs I've got PR'd) we don't tend to capitalise the first letter, I see you didn't in the first iteration of the page. Is there a reason to change this?

Reply:
I 👀  on Spaces that 99% of the time, we have:

- Capitalised the first letters of bullet points
- Capitalised the first letters in the description column of tables 

However, you have pointed out capitalisation within a **[definition list](https://textile-lang.com/doc/definition-lists#:~:text=More%20about%3A-,Definition%20lists,-1.%20The)** and I can see that in **[Spaces](https://ably.com/docs/spaces#sdk)** we do not capitalise these. However, as you may have guessed, I prefer to capitalise ALL for consistency. Happy to back down, though @GregHolmes  :) 

@m-hulbert WDYT?

Reply:
I'd keep it consistent with other definition lists for now (lower case) and we can discuss creating a new rule in the new style guide going forwards.

Reply:
Fixed and rebased!